### PR TITLE
Apekun tasks

### PR DESCRIPTION
**Pull request by zasferg**

### CODE REVIEW

**Review by MikitaTsiarentsyeu — CHANGES_REQUESTED** [review]
A function is good, float-based calculation for money is not, use Decimal instead.

Also I would like you to move your file to a specific folder with your name on it.

**Comment by zasferg** [comment]
Done

File: Apekun_Dmitry_classwork_15.03.2021/lesson2.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+c = int(input("insert c: \n"))
+
+if b < 0 and c < 0:
+    print(a,"x^2+",b*(-1), "x+",c*(-1), "= 0")
+elif c < 0 and b > 0:
+    print(a,"x^2-",b,"x-",c*(-1), "= 0")
+elif b < 0 and c > 0 :
+    print(a,"x^2+",b*(-1), "x-", c, "= 0")
+else:
+    print(a,"x^2-",b, "x+", c, "= 0")
```
Comment (MikitaTsiarentsyeu) [inline Apekun_Dmitry_classwork_15.03.2021/lesson2.py:14]: not optimal flow control, correct though

File: Apekun_Dmitry_homework_14.03.2021/hometask1.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+def summary(depo,percent,year):
+
+    month = year*12
+    perc_per_month = percent/100/12
+
+    for i in range(month):
+
+        per_month = decimal.Decimal(depo)*decimal.Decimal(perc_per_month)
+        depo+=per_month
```
Comment (MikitaTsiarentsyeu) [inline Apekun_Dmitry_homework_14.03.2021/hometask1.py:13]: a cycle is not optimal solution, there is a special formula to calculate such deposits

**Review by MikitaTsiarentsyeu — APPROVED** [review]
Good job, your marks are:
CW - 9.0
HW - 10.0

**Pull request merged**